problem_builder/management/commands/copy_deprecated_course_id.py

```diff
@@ -0,0 +1,46 @@
+import math
+import time
+from django.core.management.base import BaseCommand
+from problem_builder.models import Answer
+
+
+class Command(BaseCommand):
+    """
+    Copy content of the deprecated Answer.course_id column into Answer.course_key in batches.
+    The batch size and sleep time between batches are configurable.
+    """
+    help = 'Copy content of the deprecated Answer.course_id column into Answer.course_key in batches'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--batch-size',
+            help='The size of each batch of records to copy (default: 5000).',
+            type=int,
+            default=5000,
+        )
+        parser.add_argument(
+            '--sleep',
+            help='Number of seconds to sleep before processing the next batch (default: 5).',
+            type=int,
+            default=5
+        )
+
+    def handle(self, *args, **options):
+        batch_size = options['batch_size']
+        sleep_time = options['sleep']
+        queryset = Answer.objects.filter(course_key__isnull=True)
+        batch_count = int(math.ceil(queryset.count() / float(batch_size)))
```

Comment:
That's going to be a performance issue for large tables, isn't it? And large tables is how we got down this road in the first place.